web/html/src/manager/maintenance/ssm/system-assignment.js

```diff
@@ -0,0 +1,40 @@
+// @flow
+
+import React, {useState} from "react";
+import SpaRenderer from "core/spa/spa-renderer";
+
+import {Messages} from "components/messages";
+import {BootstrapPanel} from "components/panels/BootstrapPanel";
+
+import {SchedulePickerForm, WithMaintenanceSchedules} from "./schedule-picker";
+
+function SystemAssignment(props: {systems: string[]}) {
+  // TODO: Add proper info text
```

Comment:
Is the `TODO` still something missing? I'd either address it or drop it.

Reply:
Yes, I need some fancy text to put in here :) I'll take care of it before merging. Suggestions welcome (check out the panel header in the *Assignment from SSM* screenshot)